cddl/contrib/opensolaris/lib/libdtrace/common/dt_link.c

```diff
@@ -229,9 +230,10 @@ prepare_elf32(dtrace_hdl_t *dtp, const dof_hdr_t *dof, dof_elf32_t *dep)
 
 		for (j = 0; j < nrel; j++) {
 #if defined(__aarch64__)
-/* XXX */
-			printf("%s:%s(%d): aarch64 not implemented\n",
-			    __FUNCTION__, __FILE__, __LINE__);
+rel->r_offset = s->dofs_offset +
```

Comment:
This should be indented to one tab spot past the 'for' loop above.

Reply:
thanks